Makefile

```diff
@@ -446,6 +465,13 @@ ci-check: test
 	./test --order=rand --rng-seed=time && \
 	\
 	[ \( $$ret1 -eq 0 \) -a \( $$ret2 -eq 0 \) ]
+	@# Fail if text files contain lines ending in a space or tabs.
+	@matches=$$(egrep -Hn "[ 	]$$" $(CHECK_TRAILING_WHITESPACE_FILES)); \
+	if [ -n "$$matches" ]; then \
+		echo "Found trailing whitespace, use 'make fmt' to fix:"; \
+		echo "$$matches" | sed 's/^/  /'; \
+		exit 1; \
+	fi
```

Comment:
`ci-check` is about testing, not formatting. It mentions CI because it's handier in CI than `check` target. I wouldn't include the formatting stuff here.

In fact, I think this entire check can be removed, because CI already has a task to run `make fmt` and check Git diff for changes — so your change to `fmt` target is sufficient to make CI fail on whitespace errors.

Reply:
Yes, that's what I noticed as well and the change is already gone with a following commit. :-)

I will then remove the `skip` part on the formatting task.